payments/vipps/after-payment.md

```diff
@@ -18,214 +18,12 @@ sidebar:
 
 {% include alert-development-section.md %}
 
-## Payment Resource
-
-The payment resource and all general sub-resources can be found in the
-[core payment resources](#create-payment) section.
-
-### Create Payment
```

Comment:
Why is "Create Payment" removed? The point of having it both in the Direct/Redirect/Seamless View pages as well as in "Other Features" is that in the introductory chapters, only the most necessary parts of the request should be present, while in "Other Features" the entire request and response should be described in minute detail.